test/e2e/data/infrastructure-docker/v1alpha4/cluster-template-ipv6/cluster-with-kcp.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
What about calling this file kcp-ipv6.yaml (there is no cluster here)?

Reply:
sure, no strong opinions from me on this. fwiw, i think there are a handful of other cases like this in case anyone wants o make a separate fixup to address for style/consistency (e.g. kcp-scale-in).